.github/workflows/release.yml

```diff
@@ -1,7 +1,9 @@
-name: Release
+name: Release Rust
```

Comment:
@Shadow53 do we need to do anything with this workflow so that it publishes the `lib` package correctly, since it's now a crate in part of a workspace? Should we by excluding the python from being published somehow?

Reply:
I think you'll want to specify `path: ./crates/lib` per https://github.com/katyo/publish-crates#inputs